lib/Doctrine/ORM/Decorator/EntityManagerDecorator.php

```diff
@@ -211,6 +211,14 @@ public function flush($entity = null)
         $this->wrapped->flush($entity);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function refresh($object, $lockMode = null)
```

Comment:
`EntityManagerDecorator` is a class that is meant to be extended, its methods are meant to be overridden. Because of that, we again cannot add new parameters here without breaking downstream implementations. `func_get_arg()` it is again, sorry.